spec/models/service_template_filter_spec.rb

```diff
@@ -48,4 +48,46 @@ def build_model
       @request.miq_request_tasks.count.should eql(1)
     end
   end
+
+  context "#automate_result_include_service_template?" do
+    before do
+      TestClass = Class.new do
+        include ServiceTemplate::Filter
+        def initialize(workspace)
+          @workspace = workspace
+        end
+      end
```

Comment:
You don't need a literal constant for this spec...instead create a `let` that holds the class, use it where needed, and then you don't need an after block.

```ruby
let(:test_class) do
  Class.new do
    include ServiceTemplate::Filter
    def initialize(workspace)
      @workspace = workspace
    end
  end
end

#in the spec
expect(test_class.automate_result_include_service_template?('a', 'b')).to be_true
```